src/semigroup/object.js

```diff
@@ -1,11 +1,12 @@
 import { Semigroup } from '../semigroup';
 import propertiesOf from 'object.getownpropertydescriptors';
+import { cacheGetters } from '../stable';
 
-const { assign, getPrototypeOf } = Object;
+const { assign, getPrototypeOf, keys } = Object;
 
 Semigroup.instance(Object, {
   append(o1, o2) {
-    let properties = assign({}, propertiesOf(o1), propertiesOf(o2));
+    let properties = assign({}, propertiesOf(o1), cacheGetters(propertiesOf(o2)));
```

Comment:
Question: Why only do this to the appended properties, and not the original object properties? 

```js
let result = append({get one() { return new Object();}}, {get two() { return new Object(); }});
result.one === result.one //=> false
result.two === result.two //=> true
```
seems like a potentially surprising outcome.

Reply:
If we wrap the original object, then we might be wrapping getters twice. Is that ok?

Reply:
We're almost certainly going to end up with that situation of multi-wrapping anyway, especially if we take the result of an append and then append it to something else.

For example, that would happen with a reduction:

```js
Monoid.reduce(Object, [a, b, c, d])
```
will wrap `b`'s getters once, `c`'s getters twice, and `d`'s getters thrice.

One way would be to check if the getter is stable by adding an `isStable` attribute, or a `[Stable]` symbol to the `get` function, and not wrapping it otherwise.

Reply:
This is what I was thinking. Should we do it now?

Reply:
👍

Reply:
That way, we could have bake it right into `stable`. If it's already stable, it just passes it by.